source/localizable/tutorial/autocomplete-component.md

```diff
@@ -15,7 +15,8 @@ As before, this creates a Handlebars template
 The Handlebars template looks like this:
 
 ```app/templates/components/filter-listing.hbs
-City: {{input value=filter key-up=(action 'autoComplete')}} 
+City: {{input value=filter key-up=(action 'autoComplete')}}
```

Comment:
How about this change in the actual app? https://github.com/emberjs/super-rentals/pull/17

If that's good, then I'll update this PR for the guide to reflect the app.